api_tests/tests/siren_schema.ts

```diff
@@ -24,7 +30,7 @@ async function assertValidSirenDocument(
     const swapResponse = await alice.cnd.fetch(selfLink);
     const swapEntity = swapResponse.data as siren.Entity;
 
-    expect(swapEntity).toMatchSchema(sirenJsonSchema);
+    expect(swapEntity).toMatchSchema(sirenSwapJsonSchema);
```

Comment:
There is a slight mixup here :)

This line used to validate that the whole response is valid Siren, ignoring the properties of the specific entity. We should keep that validation :)